Source/WPF/MyMoney/Dialogs/LoanDialog.xaml.cs

```diff
@@ -70,6 +71,10 @@ public LoanDialog(MyMoney money, Account a)
                     InsertAccount(oa);
                 }
             }
+
+            List<string> currencies = new List<string>(Enum.GetNames(typeof(RestfulWebServices.CurrencyCode)));

```

Comment:
Pity these don't have tooltips...

Reply:
I added the tooltips and templatized the style, and pushed this changes so I can merge the PR.